.ci/install_runtime.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+source ${TESTS_CI_DIR}/lib.sh
+
+runtime_dir="${GOPATH}/src/github.com/clearcontainers/runtime"
+runtime_config_path="/etc/clear-containers"
+runtime_config_file="configuration.toml"
```

Comment:
Nit: this could be `config/configuration.toml` to avoid having to add the `config/` in the `cp` below (but I guess you did it this way to make the message clearer to users).